ci/scripts/github/common.sh

```diff
@@ -61,7 +61,17 @@ export SCCACHE_REGION="us-east-2"
 export SCCACHE_IDLE_TIMEOUT=32768
 #export SCCACHE_LOG=debug
 
-export CMAKE_BUILD_ALL_FEATURES="-DCMAKE_MESSAGE_CONTEXT_SHOW=ON -DMORPHEUS_CUDA_ARCHITECTURES=60;70;75;80 -DMORPHEUS_BUILD_BENCHMARKS=ON -DMORPHEUS_BUILD_EXAMPLES=ON -DMORPHEUS_BUILD_TESTS=ON -DMORPHEUS_USE_CONDA=ON -DMORPHEUS_PYTHON_INPLACE_BUILD=OFF -DMORPHEUS_PYTHON_BUILD_STUBS=ON -DMORPHEUS_USE_CCACHE=ON"
+# Set the build flags
+BUILD_DIR=${BUILD_DIR:-build}
+_FLAGS="-B ${BUILD_DIR} -G Ninja -DCMAKE_MESSAGE_CONTEXT_SHOW=ON -DMORPHEUS_CUDA_ARCHITECTURES=RAPIDS"
+_FLAGS="${_FLAGS} -DMORPHEUS_USE_CONDA=ON -DMORPHEUS_USE_CCACHE=ON"
+_FLAGS="${_FLAGS} -DMORPHEUS_PYTHON_INPLACE_BUILD=OFF -DMORPHEUS_PYTHON_BUILD_STUBS=ON"
+_FLAGS="${_FLAGS} -DMORPHEUS_BUILD_BENCHMARKS=ON -DMORPHEUS_BUILD_EXAMPLES=ON -DMORPHEUS_BUILD_TESTS=ON "
```

Comment:
1. Split each on separate line.
2. Use bash arrays when appending like this. Example in `run_conda_build.sh`